drivers/gles2/shaders/scene.glsl

```diff
@@ -97,7 +97,7 @@ uniform float light_bias;
 uniform float light_normal_bias;
 #endif
 
-uniform int view_index;
+uniform lowp int view_index;
```

Comment:
This is going to conflict with https://github.com/godotengine/godot/pull/50237

Reply:
I just needed to have the same precision in the vertex and fragment shaders to avoid getting an error from MetalANGLE. I'll eventually rebase this and keep the `highp` from that PR when it's already merged.

Reply:
Makes sense!